Lesson30/Kadir/my-app/src/components/Exercise10Double/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const Exercise10Double = () => {
+  const handleDoubleClick = () => {
+    alert('Button was double-clicked!');
+  }
+
+  return (
+    <button onDoubleClick={handleDoubleClick}>Double Click Me</button>
```

Comment:
There's no `onDoubleClick` event listener, unfortunately. What you need to do is this:
```suggestion
    <button onClick={handleDoubleClick}>Double Click Me</button>
```